### PR TITLE
DAL init

### PR DESCRIPTION
**Pull request by Leeyaoi**

### CODE REVIEW

File: InnoViber.DAL/User.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿namespace InnoViber.DAL
+{
+    public class User
+    {
+        public int Id { get; set; }
```
Comment (pavelGagarin0870) [inline InnoViber.DAL/User.cs]: Better to use Guid instead off Int

File: InnoViber.DAL/User.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        public string Name { get; set; } = "";
```
Comment (pavelGagarin0870) [inline InnoViber.DAL/User.cs]: string.Empty instead of ""

File: InnoViber.DAL/Chat.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿namespace InnoViber.DAL
+{
+    public class Chat
+    {
+        public int ID { get; set; }
```
Comment (pavelGagarin0870) [inline InnoViber.DAL/Chat.cs]: ID )

File: InnoViber.DAL/Message.cs [hunk 4]
```diff
@@ -0,0 +1,12 @@
+﻿namespace InnoViber.DAL
+{
+    public class Message
+    {
+        public int Id { get; set; }
+        public DateTime Date {  get; set; }
+        public string Text { get; set; } = "";
+        public int Status { get; set; }
+        public User User { get; set; } = new();
```
Comment (pavelGagarin0870) [inline InnoViber.DAL/Message.cs]: Make nullable and remove new();

Comment (pavelGagarin0870) [reply InnoViber.DAL/Message.cs]: Nullable because we may not use Includes and Loading

**Review by pavelGagarin0870 — CHANGES_REQUESTED** [review]

**Review by pavelGagarin0870 — APPROVED** [review]

File: InnoViber.DAL/Chat.cs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+﻿namespace InnoViber.DAL
```
Comment (pavelGagarin0870) [inline InnoViber.DAL/Chat.cs:1]: File scoped namespaces

File: InnoViber.DAL/Message.cs [hunk 6]
```diff
@@ -0,0 +1,14 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace InnoViber.DAL
```
Comment (pavelGagarin0870) [inline InnoViber.DAL/Message.cs:3]: File scoped namespaces

File: InnoViber.DAL/User.cs [hunk 7]
```diff
@@ -0,0 +1,14 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace InnoViber.DAL
```
Comment (pavelGagarin0870) [inline InnoViber.DAL/User.cs:3]: File scoped namespaces

**Review by pavelGagarin0870 — APPROVED** [review]

**Review by MaliyOna — APPROVED** [review]

**Pull request merged**